doc/using-logging.rst

```diff
@@ -48,11 +48,55 @@ supports comments and substitutions.
 It can contain multiple ``<logger>`` elements,
 each of which can have any number of :ref:`handler elements <logging-handlers>`.
 
-
 .. zconfig:: ZConfig.components.logger
     :file: logger.xml
     :members: logger
 
+.. highlight:: xml
+
+Examples
+--------
+
+Here's the configuration we looked at above. It configures the root
+(unnamed) logger with one handler (``<logfile>``), operating at the INFO level::
+
```

Comment:
A previous '.. highlight:: xml' was taking care of it. But now the configs have been moved to separate files, included with 'literalinclude' and ':language: xml'.